### PR TITLE
Math magicians: components: Styles for calculator

### PR DESCRIPTION
**Pull request by kalbek**
## Changes made in this PR:

- [ ] Delete all the boilerplate from Create React App 
- [ ] Create a components directory and calculator.js file
- [ ] Develop the UI of the calculator using JSX and CSS
- [ ] Used 1 child component (```<Calculator />```) in App.js

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @kalbek 👍, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive PR. **Pull Request should have a short summary describing introduced changes in more detail.** 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './calculator.css';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (codecaiine) [inline src/index.js:10]: - [ ] According to project requirements,  Once the Calculator is ready, you should import it and use it in your top-level component: **App.js**  in React case, the main component, the one that represents your entire application/web-site. “App.js” would usually refer to just that, your main component. Please fix it 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @kalbek , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — APPROVED** [review]
Hi @kalbek ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Added Components ✔️ 
- Good React Setup ✔️ 
- Removed boilerplate ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**